File: FastClass/salida/java/test6.java
public class test6 extends Padre { 
	String property;
	int read;
	Libro read_write;
	Cuaderno write_constructor;

	public test6(String property, Cuaderno write_constructor) {
		this.property = property;
		this.write_constructor = write_constructor;
	}
	public String getProperty(){
		return this.property;
	}
	public void setProperty(String property){
		this.property = property;
	}
	public int getRead(){
		return this.read;
	}
	public Libro getRead_write(){
		return this.read_write;
	}
	public void setRead_write(Libro read_write){
		this.read_write = read_write;
	}
	public void setWrite_constructor(Cuaderno write_constructor){
		this.write_constructor = write_constructor;
	}

}

File: FastClass/salida/php/test6.php
<?php 
class test6 extends Padre { 
	public $property;
	public $read;
	public $read_write;
	public $write_constructor;

	public __construct($property, $write_constructor) {
		$this->property = $property;
		$this->write_constructor = $write_constructor;
	}
	public function getProperty(){
		return $this->property;
	}
	public function setProperty($property){
		$this->property = $property;
	}
	public function getRead(){
		return $this->read;
	}
	public function getRead_write(){
		return $this->read_write;
	}
	public function setRead_write($read_write){
		$this->read_write = $read_write;
	}
	public function setWrite_constructor($write_constructor){
		$this->write_constructor = $write_constructor;
	}

}
?>
